.github/workflows/java8_integration_tests_ft.yml

```
@@ -53,6 +53,8 @@ jobs:
           ALLUXIO_DOCKER_FORK_COUNT=1 \
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE=dora/tests/integration \
```

```suggestion
          ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE=\!assembly/client,\!assembly/server,\!webui \
```
shouldn't this be identical to the one in java8_integration_tests.yml?  otherwise we are back to the situation before your PR https://github.com/Alluxio/alluxio/pull/18313 where mvn compile project list == mvn test project list

OR did we have the wrong explanation for the error because the old way was using `dora/tests` in the project list, which is an invalid module? or in other words, we thought the bug was in the maven project list, but the actual bug was using `dora/tests`. if this is true and this github action is working with compile == `dora/test/integration` and test == `dora/test/integration`, then we don't actually need to separate the mvn arguments!

---

i addressed my own comment in a new PR: https://github.com/Alluxio/alluxio/pull/18392